subsys/nrf_security/src/drivers/cracen/cracenpsa/src/kmu.c

```diff
@@ -211,7 +218,13 @@ int cracen_kmu_prepare_key(const uint8_t *user_data)
 int cracen_kmu_clean_key(const uint8_t *user_data)
 {
 	(void)user_data;
+	/* The kmu_push_area is guarded by the symmetric mutex since it is the most common use case.
+	 * Here the decision was to avoid defining another mutex to handle the push buffer for the rest
+	 * of the use cases.
+	 */
+	nrf_security_mutex_lock(cracen_mutex_symmetric);
```

Comment:
I don't think we need to lock the mutex here (normally called in sx symcrypt free functions?)